portfolio/src/main/webapp/scripts/auth.js

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Handles the Auth API requests
+ */
+
+// Variable to keep track of the auth status in the current session
+let verificationStatus = false; /** @type { bool } */
```

Comment:
```suggestion
/** @type { boolean } */
let verificationStatus = false;
```
Typically JSDoc goes above the variable declaration. Also, I think the formal type is "boolean".